101-cosmosdb-gremlin/azuredeploy.json

```diff
@@ -197,18 +197,7 @@
                         "includedPaths": [
                             {
                                 "path": "/*",
-                                "indexes": [
-                                    {
-                                        "kind": "Range",
-                                        "dataType": "number",
-                                        "precision": -1
-                                    },
-                                    {
-                                        "kind": "Range",
-                                        "dataType": "string",
-                                        "precision": -1
-                                    }
-                                ]
+                                "indexes": []
```

Comment:
Can we just remove the indexes property since it's empty?

Reply:
We actually would prefer these remain empty arrays's since they might be used for customers implementing a custom index policy and/or adding a spatial index type. The empty property will make it easier for customers to visually know where to put them.

Thanks Brian.

Reply:
The problem with empty properties on a put is it will remove any values already set, which is a common mistake users make. GET is showing the correct property value since it shows what is...  Putting a valid index in is fine, but empty has an explicit sometimes unexpected meaning.

Reply:
I'm not sure I follow your logic. 

If someone changes the properties of a deployed resource outside of the template or does not update the template to reflect the changes made elsewhere, then that would seem to be the actual problem, not the empty array which reflects what was deployed initially. This would be true for any property, there or not.

That said I'm more interested in getting this merged than speculating on this so I will update as you suggested with a new commit.